src/configSchema.ts

```diff
@@ -184,4 +203,11 @@ export const VALIDATION_SCHEMA = z
 	.refine(
 		(config) => config.matchMode === MatchMode.SAFE || config.linkDir,
 		ZodErrorMessages.needsLinkDir,
+	)
+	.refine(
+		(config) =>
+			config.linkDir &&
+			config.dataDirs &&
+			isPathInside(config.linkDir, config.dataDirs),
```

Comment:
will the &&s here cause an error if you don't have a link dir or a data dir?

Reply:
That's interesting, yes it does.

Reply:
Please hold. :P

Reply:
Tested fine now with undefined.